android/festago/data/src/main/java/com/festago/festago/data/repository/FakeFestivalRepository.kt

```diff
@@ -27,18 +32,24 @@ class FakeFestivalRepository @Inject constructor() : FestivalRepository {
         val notNullSize = size ?: DEFAULT_SIZE
         val notNullLastFestivalId = lastFestivalId ?: DEFAULT_LAST_FESTIVAL_ID
```

Comment:
```suggestion
    override suspend fun loadFestivals(
        schoolRegion: SchoolRegion?,
        festivalFilter: FestivalFilter?,
        lastFestivalId: Long?,
        lastStartDate: LocalDate = lastFestivalId ?: DEFAULT_LAST_FESTIVAL_ID,
        size: Int = DEFAULT_SIZE,
    ): Result<FestivalsPage> {
```
과 같이 하지 않은 이유가 있었을까요?

Reply:
lastStartDate 에 lastFestivalId 를 넣는 이유가 무엇인가요..?

Reply:
아 오타가 났네요
```
    override suspend fun loadFestivals(
        schoolRegion: SchoolRegion?,
        festivalFilter: FestivalFilter?,
        lastFestivalId: Long = lastFestivalId ?: DEFAULT_LAST_FESTIVAL_ID,
        lastStartDate: LocalDate?,
        size: Int = DEFAULT_SIZE,
    ): Result<FestivalsPage> {
```
이겁니다

Reply:
가능한 문법인가요..??
default value 에서 변수를 사용하거나 lastFestivalId 가 null 인 상황이 불가능합니다!

<img width="487" alt="image" src="https://github.com/woowacourse-teams/2023-festa-go/assets/108349655/27f8c106-1a92-4784-84bd-495eb8c63f4d">

Reply:
또한 overriding function 은 허용되지 않는다고 하네요

Reply:
괜찮습니다!!